libbeat/common/cli/password.go

```diff
@@ -75,5 +75,10 @@ func env(p string) (string, error) {
 		return "", errors.New("env variable name is needed when using env: password method")
```

Comment:
I noticed the error uses inconsistent terminology "env var" vs "environment variable". I prefer the latter like what you have added in the new error.

Reply:
good point, I will replace that.